File: 12/Problem1.c
#include<stdio.h>

void ReverseDigit(int iNo)
{
    int iDigit = 0;
    if(iNo < 0 )
    {
        iNo = -iNo;
    }
        while(iNo != 0)
        {
            iDigit = iNo % 10;
            printf("%d",iDigit);
            iNo = iNo / 10;   
        }
        printf("\n");
        
}
int main()
{
    int iValue = 0;
    
    printf("Enter the number : \n");
    scanf("%d",&iValue);
    ReverseDigit(iValue);
    
    return 0;
}

File: 17/problem1.java
import java.util.*;

class DigitX
{
    public int SumDigit(int iNo)
    {
        int iDigit = 0;
        int iSum = 0;

        while(iNo != 0)
        {
            iDigit = iNo % 10 ;
            iSum = iSum + iDigit;
            iNo = iNo/10;
        }
        return iSum;
    }
}
class problem1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        DigitX dobj = new DigitX();

        int iValue = 0, iRet = 0;

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        iRet = dobj.SumDigit(iValue);
        System.out.println("Summation of Digits is : "+iRet);

    }
}